charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -91,6 +91,10 @@ node:
 
 serviceAccount:
   controller:
+    create: true # A service account will be created for you if set to true. Set to false if you want to use your own.
```

Comment:
Over at https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/296/files#diff-81801117ec01136c8a49bfcd42af8e104f4efad1f2daccda9da9d960eaad5279R74 i was thinking to make it like

controller.serviceAccount.create instead of
serviceAccount.controller.create

I want to be consistent so I'll change the EFS one as it hasn't been released yet.